modules/docs/src/01-overview/02-quickstart.md

```diff
@@ -0,0 +1,137 @@
+---
+sidebar_label: Quick Start
+title: Quick Start
+---
+
+Below is a quick example of smithy4s in action. This page does not give very much explanation or detail. For more information on various aspects of smithy4s, read through the other sections of this documentation site.
+
+## project/plugins.sbt
+
+Add the smithy4s codegen plugin to your build.
+
+```scala
+addSbtPlugin("com.disneystreaming.smithy4s" % "smithy4s-sbt-codegen" % "@VERSION@")
+```
+
+## build.sbt
+
+Enable the plugin in your project, add the smithy and http4s dependencies.
+
+```scala
+import smithy4s.codegen.Smithy4sCodegenPlugin
+
+ThisBuild / scalaVersion := "@SCALA_VERSION@"
+
+val example = project
+  .in(file("modules/example"))
+  .enablePlugins(Smithy4sCodegenPlugin)
+  .settings(
+    libraryDependencies ++= Seq(
+      "com.disneystreaming.smithy4s" %% "smithy4s-http4s" % smithy4sVersion.value,
+      "com.disneystreaming.smithy4s" %% "smithy4s-http4s-swagger" % smithy4sVersion.value,
```

Comment:
So technically, `smithy4s-http4s-swagger` brings in `smithy4s-http4s` transitively, should we still include both here? My gut says we should leave both.. but wanted to get other thoughts

Reply:
Personally I don't care much about explicit dependencies, transitive is fine for me - I'd leave just the one that takes in all the others

Reply:
For the sake of this doc, I'd rather keep both.